backend/ClassSchedulerBackend/models.py

```diff
@@ -0,0 +1,168 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class DayTime(models.Model):
+    class Meta:
+        db_table = 'day_time'
+
+    day_time_id = models.AutoField(primary_key=True)
+    time = models.CharField(default=None, max_length=100)
+
+
+class WeekDay(models.Model):
+    class Meta:
+        db_table = 'week_day'
+
+    week_day_id = models.AutoField(primary_key=True)
+    week_day = models.CharField(default=None, max_length=100)
+
+
+class Classroom(models.Model):
+    class Meta:
+        db_table = 'classroom'
+
+    classroom_id = models.AutoField(primary_key=True)
+    capacity = models.IntegerField(default=None)
+
+
+class Semester(models.Model):
+    class Meta:
+        db_table = 'semester'
+
+    semester_id = models.AutoField(primary_key=True)
+    year = models.IntegerField(default=None)
+    name = models.CharField(default=None, max_length=100)
+    duration_weeks = models.IntegerField(default=None)
+
+
+class ParameterData(models.Model):
+    class Meta:
+        db_table = 'parameter_data'
+
+    parameter_id = models.AutoField(primary_key=True)
+    approved = models.BooleanField(default=False)
+    requirement = models.BooleanField(default=False)
+    score = models.IntegerField(default=None)
+
+
+class TimeSlot(models.Model):
+    class Meta:
+        db_table = 'time_slot'
+
+    time_slot_id = models.AutoField(primary_key=True)
+    week_day_obj = models.ForeignKey(WeekDay, null=True, default=None, on_delete=models.CASCADE)
```

Comment:
I think this should be `week_day_id` as per ERD. Can you please make sure all columns have the same name as in ERD?